service/controller/v14/version_bundle.go

```diff
@@ -9,8 +9,8 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "aws-operator",
-				Description: "Add your changes here.",
-				Kind:        versionbundle.KindChanged,
+				Description: "Added Route53 support for BYOC accoutns.",
+				Kind:        versionbundle.KindAdded,
```

Comment:
Is this the only change?

Reply:
I don't know of any other changes and k8scc didn't change.

Also just a typo on accounts.